lua/lspconfig/tailwindcss.lua

```diff
@@ -92,12 +92,11 @@ configs[server_name] = {
       end
     end,
     root_dir = function(fname)
-      return util.root_pattern("tailwind.config.js", "tailwind.config.ts")(fname) or util.root_pattern(
-        "postcss.config.js",
-        "postcss.config.ts"
-      )(fname) or util.find_package_json_ancestor(fname) or util.find_node_modules_ancestor(fname) or util.find_git_ancestor(
-        fname
-      )
+      return util.root_pattern("tailwind.config.js", "tailwind.config.ts")(fname)
+        or util.root_pattern("postcss.config.js", "postcss.config.ts")(fname)
```

Comment:
in nvim core we are almost certainly going to use single quotes, would be good to align with that in lspconfig

Reply:
I'll change the default formatting template, it's been on my to-dos for awhile now :)

Reply:
done